semver/less_than_range_test.ts

```diff
@@ -169,3 +170,22 @@ Deno.test("lessThanRange() checks if the SemVer is less than the range", async (
     });
   }
 });
+
+Deno.test("lessThanRange() handles not equals operator", () => {
+  const version = {
+    major: 1,
+    minor: 0,
+    patch: 0,
+    prerelease: [],
+    build: [],
+  };
+  const range = [[{
+    operator: "!=" as unknown as Operator,
+    major: 1,
+    minor: 0,
+    patch: 0,
+    prerelease: [],
+    build: [],
+  }]];
+  assertEquals(lessThanRange(version, range), true);
```

Comment:
This result feels wrong to me. I'd suggest skipping this test case for now

Reply:
I think we should not skip test cases if they yield unexpected results but fix them instead. What would the proper result be?

Reply:
Would it make sense to create a PR to remove the `!=` straight away as a bug fix before landing this PR? This probably solves lots of headaches with intermediate patches.

Reply:
That will be a breaking change and we currently don't have a plan for it.

Reply:
I think this bug could be fixed by modifying the `!=` case to `return cmp === 0;` instead of `return false;` in `greaterThanComparator()` and `lessThanComparator()`. Then it will return the expected `false`.